source/manual/govuk-notify.html.md

```diff
@@ -40,6 +40,10 @@ for each environment):
   example might be to receive an email when a scheduled document on GOV.UK has
   been automatically published.
 
+- **GOV.UK Public**
+
+  This service is used by [Feedback](https://github.com/alphagov/feedback). It's used to send public e-mails to users who have clicked on a survey banner in order for us to send them a link to the survey.
```

Comment:
Suggest changing this to `emails` for consistency

Reply:
👍